### PR TITLE
make changes on milliqanScheduler, so it can work without plotter class

### PR DESCRIPTION
**Pull request by CollinKa**
Sometimes we only want to do a quick analysis without making plots.  I make changes on milliqanScheduller so that it can work without a plotter.

### CODE REVIEW

File: Run3Detector/analysis/utilities/milliqanScheduler.py [hunk 1]
```diff
@@ -21,7 +21,10 @@ def __init__(self, inputs, cuts=None, plotter=None):
     def setEvents(self, events):
         self.events = events
         self.cuts.events = self.events
-        self.plotter.events = self.events
+        try:
+            self.plotter.events = self.events
+        except AttributeError as error:
+            print("please be awared that plotter is not being used now")
```
Comment (mlj5j) [inline Run3Detector/analysis/utilities/milliqanScheduler.py:27]: Spelling error:  Change "awared" to "aware".

Comment (mcitron) [reply Run3Detector/analysis/utilities/milliqanScheduler.py:27]: Is an exception really the best way to deal with this?

Comment (CollinKa) [reply Run3Detector/analysis/utilities/milliqanScheduler.py]: what about if self.plotter != None:  self.plotter.events = self.events

> Is an exception really the best way to deal with this?

**Review by mlj5j — APPROVED** [review]
Looks fine to me.

**Pull request merged**